modules/serverless-rn-1-9-0.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-9-0_{context}"]
+//update the <version> to match the filename
+
+= Release Notes for Red Hat {ServerlessProductName} 1.9.0
+
+* {ServerlessProductName} now uses Knative Serving 0.15.0.
+* {ServerlessProductName} now uses Knative Serving Operator 0.15.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.15.0.
+* {ServerlessProductName} uses Knative Eventing 0.15.0.
+* {ServerlessProductName} now uses Knative Eventing Operator 0.15.0.
+* {ServerlessProductName} now uses Kourier 0.15.0.
+
+[id="new-features-1-9-0_{context}"]
+== New features
+
+* {ServerlessProductName} now supports some integrated Red Hat OpenShift Service Mesh features, including enabling sidecars, and JSON Web Token (JWT) authentication. Supported features are documented in the _Integrations_ section.
+
+[id="fixed-issues-1-9-0_{context}"]
+== Fixed issues
+
+* In previous versions of {ServerlessProductName}, deleting a sink before deleting the SinkBinding connected to it caused a hanging issue that required removing a finalizer from the SinkBinding to resolve. This issue is fixed in {ServerlessProductName} 1.9.0.
+
+[id="known-issues-1-9-0_{context}"]
+== Known issues
```

Comment:
It would be good to document this as a known issue: https://issues.redhat.com/browse/SRVKE-546